examples/tutorials/advanced/websockets-example-MNIST/Federated learning with websockets and federated averaging.ipynb

```diff
@@ -1,15 +1,21 @@
 {
  "cells": [
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
```

Comment:
Should be markdown not code: have a look at your notebook in jupyter to make sure it prints nicely, here is how it looks like currently: https://github.com/OpenMined/PySyft/blob/039798359758a5f02d841c5d2c15fe676318a0c4/examples/tutorials/advanced/websockets-example-MNIST/Federated%20learning%20with%20websockets%20and%20federated%20averaging.ipynb

Reply:
I might have accidently did that while on pycharm changing it asap

Reply:
Install the markdown module for pycharm :)
Put you should prefer jupyter notebook I'd say :)